less/index.less

```diff
@@ -57,3 +57,156 @@ html, body {
     height: 100%;
 }
 
+@import url(https://fonts.googleapis.com/css?family=BenchNine:700);
+@import url(https://fonts.googleapis.com/css?family=Open+Sans+Condensed:300);
+
+*{
+	box-sizing:border-box;
+}
+
+html,body{
+  height:100%; 
+  background: linear-gradient(left, #BFE6FC, white,#BFE6FC);
+  background: linear-gradient(to right, #BFE6FC, white,#BFE6FC);
+}
+
+.container{
+	padding: 30px 30px;
+  max-width:100%;
+	height:100%;
+	display: flex;
+	flex-direction: column;
+}
+
+// Key Words Colors and Sizes
+
+.word {
+  display:block;
+  text-align:justify;
+  font-family: "BenchNine";
+  line-height:0.8;
+  text-transform:uppercase;
+  letter-spacing:0px;
+}
+
+.word1, .word2 {color: palevioletred}
+.word3, .word4 {color:#4E8AB4}
+.word5, .word6{color: black}
+.word7 {color:#6B7B8A}
+.word8, .word9{color:#8B8091}
+.word10, .word11{color:#B68FA2}
+.word12, .word13{color:#F7C711}
+
+.word1{font-size:6.3em}
+.word2{font-size:7.1em}
+.word3{font-size:4.8em}
+.word5{font-size:4.8em}
+.word7{font-size:4.5em}
+.word8{font-size:4.8em}
+.word12{font-size:6.2em}
+
+.contact p {
+	margin: 15px;
+	padding-bottom: 20px;
+}
+.info {
+	font-size: 1.2em;
+	font-weight: bold;
+	// font-style: italic;
+}
+
+// MAIN CONTENT 
+
+.container .main-content {
+margin: 80px;
```

Comment:
i would make a media query here for mobile and take off the left and right padding on main content so it fits a lil better